### ISSUE TITLE
Safari bug: Hovering over links resets scroll position

### ISSUE THREAD

**Issue by Sam0711er** [hfg-gmuend/openmoji-website#15]
When you hover over a link, the page scrolls back to the top.

How to reproduce:
1. Open [http://openmoji.org/about.html](http://openmoji.org/about.html)
2. Scroll down
3. Hover over a link (like Twitter Icon below a profile)
4. The site scrolls back to the top

Used configuration:
Browser: Safari 12.0.3 (14606.4.5)
System: macOS Mojave 10.14.3 (18D109)
Enabled plugins: None

**Comment by b-g** [comment]
@PaulRaschke Could you have a look on that bug? This is really a big problem, as you can't even browse with Safari the library site :( https://hfg-gmuend.github.io/openmoji-website/library.html

**Comment by PaulRaschke** [comment]
Hi @b-g,

unfortunately I don't have Safari (no MacOS) to have a look on the Bug and I couldn't reproduce the Bug in any other popular browser. I ran Safari on a Virtual Machine but wasn't able to fix the Bug in short time and debugging really was a pain because the Virtual Machine was laggy and randomly crashed sometimes. I only discovered that I could hover the Icons without a problem, when I removed the "animation" (e.g. the change in opacity in the iconlink class) but the scrolling was still jumpy.
[This](https://github.com/iamceege/tooltipster/issues/140) bug report depicts a similiar Bug as it seems.
Can anybody else try to fix the Bug?

**Comment by Sam0711er** [comment]
https://github.com/hfg-gmuend/openmoji-website/pull/18

**Comment by brandonweiss** [comment]
This might be the same bug I’m running into [here](https://stackoverflow.com/questions/55268747/in-safari-hovering-over-svg-elements-causes-a-scrolled-element-to-reset-its-scr/)? Unfortunately I have not yet figured out how to fix it in a way that doesn’t break the full width/height layout 😞

**Comment by b-g** [comment]
@brandonweiss Thanks for the pointer! We'll check ... 🤞that we can figure it out for Safari!

**Comment by brandonweiss** [comment]
I found that it seemed to somehow be related to CSS grid and SVGs inside of anchor tags. When I switched from CSS grid to flexbox, it fixed the issue. 😞

**Comment by Sam0711er** [comment]
Hey @brandonweiss, thx for pointing that out!
We temporarily fixed this issue [here](https://github.com/hfg-gmuend/openmoji-website/pull/18) by removing `height: 100vh` from the body. However, this killed lazy loading for us (which shouldn't be too big of a problem to fix now)

**Comment by Sam0711er** [comment]
Hey @b-g @brandonweiss!
I got in touch with the WebKit team and asked for their help. They asked me to verify this issue with Safari Technology Preview (The Public Beta of Safari) and it seems that this is indeed a Safari bug that has been resolved there. Therefore, the problem should solve itself as soon as the new version of Safari is released.

[https://bugs.webkit.org/show_bug.cgi?id=197189](https://bugs.webkit.org/show_bug.cgi?id=197189)
